tests/testthat/mock_louper.bat

```diff
@@ -1,5 +1,66 @@
 @echo off
 
 echo "-----------RUNNING MOCK LOUPER.BAT EXECUTABLE--------------"
-echo Arguments passed to the script: %*
+
+setlocal enabledelayedexpansion
+
+set "force=false"
+
+REM Check if the correct number of arguments is provided
+if "%~1" neq "create" (
+    echo Error: First argument must be 'create'
+    goto :eof
+)
+shift
+
+REM Parse and validate the input and output options
+set "input="
+set "output="
+
+:parse_args
+if "%~1" equ "" goto :check_paths
+
+if /i "%~1" equ "--input" (
```

Comment:
The `"%~1" will strip any surrounding quotes around the argument, but it seems to do wonky things depending on if there is a space between the equal sign.  `--output="some path"` vs `--output = "some apth"`